README.md

```diff
@@ -1,42 +1,70 @@
 # Breast density classification with deep convolutional neural networks
 ## Introduction
-This is an implementation of the model used for breast density classification as described in our paper ["Breast density classification with deep convolutional neural networks"](https://arxiv.org/pdf/1711.03674.pdf). The implementation allows users to get the breast density prediction by applying one of our pretrained models: a histogram-based model or a multi-view CNN. Both models act on screening mammography exams with four standard views. As a part of this repository, we provide a sample exam (in `images` directory).
+This is an implementation of the model used for breast density classification as described in our paper ["Breast density classification with deep convolutional neural networks"](https://arxiv.org/pdf/1711.03674.pdf). The implementation allows users to get the breast density prediction by applying one of our pretrained models: a histogram-based model or a multi-view CNN. Both models act on screening mammography exams with four standard views. As a part of this repository, we provide a sample exam (in `images` directory). The models are implemented in both TensorFlow and PyTorch.
 
 ## Prerequisites
 
-* Python (3.6), TensorFlow (1.5.0), NumPy (1.14.3), SciPy (1.0.0).
-* NVIDIA GPU (we used Tesla M40).
```

Comment:
Please keep the version of NVIDIA GPU in the readme.md

Reply:
GPU usage is now optional, but I can add it back if you think it’s necessary.